platform/security/cas/security-cas-client/src/main/resources/OSGI-INF/blueprint/casfilters.xml

```diff
@@ -73,10 +73,6 @@
                 <ref component-id="casAuthenticationFilter"/>
             </list>
         </argument>
-        <cm:managed-properties persistent-id="ddf.security.cas"
-          update-strategy="container-managed"/>
-        <property name="serverName"
-                  value="${org.codice.ddf.external.protocol}${org.codice.ddf.external.hostname}:${org.codice.ddf.external.port}${org.codice.ddf.external.context}" />
```

Comment:
❓ Should this be removed or use the internal properties instead?

Reply:
No this config is used to construct the service url that gets included as a query parameter when the user is redirected to CAS. That won't get rewritten by the proxy without more complicated url rewriting, so makes sense to use the external URL